miden-test-utils/src/lib.rs

```diff
@@ -0,0 +1,176 @@
+pub use vm_core::{stack::STACK_TOP_SIZE, Felt};
+
+pub mod crypto;
+mod tester;
+pub use tester::{
+    convert_to_stack, prop_randw, AdviceInputs, MerkleStore, StackInputs, Test, TestError,
+    U32_BOUND,
+};
+
+// TESTS
+// ================================================================================================
+
+#[test]
+fn simple_program() {
+    build_test!("begin push.1 push.2 add end").expect_stack(&[3]);
+}
+
+#[test]
+fn multi_output_program() {
+    let test = build_test!("begin mul movup.2 drop end", &[1, 2, 3]);
+    test.prove_and_verify(vec![1, 2, 3], false);
+}
```

Comment:
I'd remove these from here. I think this crate should be a utility crate used in other places but shouldn't actually contain any tests itself.

Reply:
Makes sense.